### ISSUE TITLE
Change Scratch Notifier Category

### ISSUE THREAD

**Issue by Samq64** [ScratchAddons/ScratchAddons#6367]
### Why this would be helpful

Scratch Notifier is in the website category which seems a little strange.

### My suggestion

Renaming the "Extension popup features" category and moving Scratch Notifier there.

**Comment by DNin01** [comment]
I think we should consider renaming that category.

**Comment by WorldLanguages** [comment]
> I think we should consider renaming that category.

That's probably the best option

**Comment by Samq64** [comment]
Would renaming the category to just "Extension features" work or would that be too vague?

**Comment by mxmou** [comment]
> Would renaming the category to just "Extension features" work or would that be too vague?

SA is an extension, so every feature is technically an extension feature.

**Comment by WorldLanguages** [comment]
Only options I can think of are "extension features" and "browser features"

**Comment by BroJac5246** [comment]
> > Would renaming the category to just "Extension features" work or would that be too vague?
> 
> SA is an extension, so every feature is technically an extension feature.

True, but because everyone knows that SA is an extension, I don't think anyone will question it, since what's the point of us clarifying something that is obvious? We'd never do it, so we logically must be referring to something else.
